src/App.tsx

```diff
@@ -15,37 +16,85 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum Way {
+  'Unknown' = 0,
+  'Info' = 'is-info',
+  'Success' = 'is-success',
+}
+
+function getSorted(obj: string[], way: Way) {
```

Comment:
The `way` parameter is typed as `Way`, but the `case` statements in the `switch` use string literals. You should use the `Way` enum values instead, like `Way.Info` and `Way.Success`.